src/installation/upgrading.md

```diff
@@ -1,43 +1,40 @@
 # Upgrading from a previous version
 
-To upgrade Erigon to a newer version when you've originally installed it via Git and manual compilation, you should follow these steps without needing to delete the entire folder:
+Updating to the latest version of Erigon gives you access to the latest features and ensures optimal performance and stability.
 
-* **Terminate your Erigon** session by pressing CTRL+C
+## General Recommendations Before Upgrade
 
-* **Navigate to your Erigon directory**
+- **Read Release Notes**: Carefully review the [Release Notes](https://github.com/erigontech/erigon/releases) for breaking changes and new features relevant to your setup.
+- **Terminate your Erigon**: End your current Erigon session by pressing `CTRL+C`.
+- **Backup**: Always back up your `datadir` before performing major upgrades.  
 
-* **Fetch the latest changes from the repository**: You need to make sure your local repository is up-to-date with the main GitHub repository. Run:
+## Upgrading your Data
 
-    ```bash
-    git fetch --tags
-    ```
+As of Erigon 3.1 Pebble Paws, the snapshot format has been updated. When upgrading from version 3.0.x, follow these steps:
 
-* **Check out** the [latest version](https://github.com/ledgerwatch/erigon/releases) and switch to it using:
+1. Backup your datadir.
+2. [Upgrade your Erigon installation](#upgrading-your-erigon-installation) whether from a binary, compiled source code, or Docker.
+3. **OPTIONAL**: Upgrading your data is not mandatory, since the new version of the Erigon binary can function with older data. However, upgrading will resolve known data-related issues.
+    1. To initiate the data upgrade, use the following command: `./build/bin/erigon snapshots reset --datadir /your/datadir`. After executing this command, Erigon will sync with newer snapshots while preserving unchanged files.
```

Comment:
This line misses the file renaming required from 3.0 to 3.1, which means users will lose most of their data.